apparmor.d/groups/browsers/firefox

```diff
@@ -134,7 +134,7 @@ profile firefox @{exec_path} flags=(attach_disconnected) {
   @{firefox_lib_dirs}/{,**}             r,
   @{firefox_lib_dirs}/*.so              mr,
   @{firefox_lib_dirs}/crashreporter     rPx,
-  @{firefox_lib_dirs}/glxtest          rPUx,
+  @{firefox_lib_dirs}/glxtest           rix,
```

Comment:
Remove this, this is rPUx on purpose, the profile is a wip.

Reply:
Hm, I'm not sure that I understand. rPUx is causing an error with aa-log (and aa-logprof) every time. How is this related with the profile being a wip? Shouldn't that rule be adjusted, at least, temporarily until a final solution is found?

Reply:
`rPUx` means it allow to transition to a profile if it exist or unconfined if it does not exist. I do this while I am working on the new `firefox-glxtest` profile. This allow me to transition to the wip profile while other can use the profile without break.